[ADD][8.0]product_by_supplir: Add active field

maurochip opened the pull request:
Added active field related to the product active field.
I think it is useful because we will only see the active
products.

Commit:
[ADD][8.0]product_by_supplir: Add active field

Added active field related to the product active field.
I think it is useful because we will only see the active
products.

pedrobaeza commented:
This affects the lines shown on Suppliers list, so looking at the inactive product will shown the list empty. You should put other name and add a filter (maybe default filter) with the field, but not call it `active`.

Commit:
[ADD][8.0]product_by_supplir: Add active filter

Added filter to see to see the active products and remove
the active field added previously.

gurneyalex commented:
I just merged #529 which fixes the travis build of 8.0. Can you rebase?
